src/uiSelectController.js

```diff
@@ -325,9 +330,9 @@ uis.controller('uiSelectCtrl',
     var isDisabled = false;
     var item;
 
-    if (itemIndex >= 0 && !angular.isUndefined(ctrl.disableChoiceExpression)) {
+    if (itemIndex >= 0 && (!angular.isUndefined(ctrl.disableChoiceExpression) || ctrl.multiple)) {
```

Comment:
@dondi could you add an extra set of brackets here to make the condition precedence clear? 

Eg. 
```
if (itemIndex >= 0 && ((!angular.isUndefined(ctrl.disableChoiceExpression) || ctrl.multiple)))
```
Or..?
```
if ((itemIndex >= 0 && (!angular.isUndefined(ctrl.disableChoiceExpression)) || ctrl.multiple))
```